bleps/src/lib.rs

```diff
@@ -594,4 +594,54 @@ pub mod asynch {
             data
         }
     }
+
+    pub trait AsyncHciConnection {
+        async fn read(&self) -> Option<u8>;
```

Comment:
1. should be reading/writing 1 byte at a time
2. should we be returning `Result<usize, SomeKindOfAsyncErrorWrapperThingie>`?